packages/components/src/internal/components/entities/EntityLineageEditModal.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { FC, memo, useCallback, useEffect, useState } from 'react';
+
+import { AuditBehaviorTypes, Utils } from '@labkey/api';
+
+import {
+    Alert,
+    capitalizeFirstChar,
+    caseInsensitive,
+    createNotification,
+    naturalSort,
+    ParentEntityEditPanel,
+    Progress,
+    QueryModel,
+    resolveErrorMessage,
+    updateRows,
+} from '../../..';
+import { EntityChoice, EntityDataType } from './models';
+import { Button, Modal } from 'react-bootstrap';
+import { getEntityNoun, getUpdatedLineageRowsForBulkEdit } from './utils';
+import { List } from 'immutable';
+import { getOriginalParentsFromSampleLineage, getSampleSelectionLineageData } from '../samples/actions';
+import { ParentEntityLineageColumns } from './constants';
+
+interface Props {
+    queryModel: QueryModel;
+    onCancel: () => void;
+    onSuccess: () => void;
+    childEntityDataType: EntityDataType;
+    auditBehavior?: AuditBehaviorTypes;
```

Comment:
I saw the `auditBehavior` prop had a default value in the `EntityLineageEditMenuItem` component. Do you think that should be set here as well as a default prop, or maybe just moved to here since this is the one that uses it?